src/main/java/com/hierynomus/mssmb2/dfs/DFS.java

```diff
@@ -0,0 +1,359 @@
+/*
+ * Copyright (C)2016 - SMBJ Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hierynomus.mssmb2.dfs;
```

Comment:
That's not as obvious as you think! :-)  Also, I don't know where the "C" comes from, but it seems kind of distracting.  But, if that's what you really are trying to stick with, then I can do that.